.github/workflows/cd.yaml

```diff
@@ -0,0 +1,34 @@
+name: Create and publish a container image
+
+on:
+  push:
+    tags:
+      - '*' # Create container image and publish it on tag creation
```

Comment:
If someone tags a WIP branch is this workflow triggered? What do you think to use the event [release](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#release) instead?

Reply:
Yes, this workflow is triggered on any tag pushed on the origin. We could use the release event, it's up to us to decide whether if we want to create the container image automatically with a tag or if we want to create the image after we have created the release on github (so, create the tag and then manually create a release from github).